sui_types/src/unit_tests/serialize_tests.rs

```diff
@@ -16,9 +16,9 @@ use super::*;
 
 // Only relevant in a ser/de context : the `CertifiedTransaction` for a transaction is not unique
 fn compare_certified_transactions(o1: &CertifiedTransaction, o2: &CertifiedTransaction) {
-    assert_eq!(o1.transaction.digest(), o2.transaction.digest());
+    assert_eq!(o1.digest(), o2.digest());
     // in this ser/de context it's relevant to compare signatures
-    assert_eq!(o1.signatures, o2.signatures);
+    assert_eq!(o1.auth_sign_info.signatures, o2.auth_sign_info.signatures);
```

Comment:
nice, I was originally puzzled re why do we compare signatures as they can be mallebale, but this is for serde tests which makes sense.